File: Java/Csharp/ChainOfResponsibility/ChainOfResponsibility/MouseHandler.cs
namespace ChainOfResponsibility
{
    class MouseHandler : AbstractHandler
    {
        public override object Handle(object request)
        {
            if (request.ToString() == "Cheese")
            {
                return $"Mouse: I love {request.ToString()}.\n";
            }
            else
            {
                return base.Handle(request);
            }
        }
    }
}

File: PHP/ChainOfResponsibility/COR1/MouseHandler.php
<?php


namespace TheRayCode\ChainOfResponsibility;


class MouseHandler extends AbstractHandler
{
    public function handle(string $request): ?string
    {
        if ($request === "Cheese") {
            return "Mouse: I'll eat the " . $request . ".<br/>";
        } else {
            return parent::handle($request);
        }
    }
}